src/List/ListItemContent.js

```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+class ListItemContent extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        icon: PropTypes.string,
+        subtitle: PropTypes.oneOfType([
+            PropTypes.element,
+            PropTypes.string
+        ]),
+        threeLine: PropTypes.bool
+    };
+
+    static defaultProps = {
+        icon: null,
+        subtitle: null,
+        threeLine: false
+    };
+
+    render() {
+        const { children, className, icon, subtitle, threeLine, ...otherProps } = this.props;
+
+        const classes = classNames('mdl-list__item-primary-content', className);
+
+        let iconElement = null;
+
+        if (icon) {
+            iconElement = (
+                <i className="material-icons mdl-list__item-avatar">{icon}</i>
+            );
+        }
+
+        let subtitleElement = null;
+
+        if (subtitle) {
+            const subtitleClassName = threeLine ? 'mdl-list__item-text-body' : 'mdl-list__item-subtitle';
+
+            subtitleElement = (
+                <span className={subtitleClassName}>
+                    {subtitle}
+                </span>
+            );
+        }
+
+        return (
+            <span className={classes}>
+                {iconElement}
```

Comment:
use the format `{icon && <Icon name={icon} />}` (with the Icon component)
btw, we have to support `mdl-list__item-icon` and `mdl-list__item-avatar`.

Might be better to use a specific component for that as well, to allow users to set something else than a Material Icon (an image for example).